sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/models/ClientEndpointType.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.webpubsub.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Defines values for ClientEndpointType
+ */
+public enum ClientEndpointType {
```

Comment:
`enum` here means no value can be added, ever (new api-version included). Make sure your intention is such.